lib/workers/repository/onboarding/pr/index.ts

```diff
@@ -128,6 +129,7 @@ If you need any further assistance then you can also [request help here](${confi
         useDefaultBranch,
       });
       logger.info({ pr: pr.displayNumber }, 'Onboarding PR created');
+      await addAssigneesReviewers(config, pr);
```

Comment:
@rarkins I think this should be done in separate pr.

Reply:
Yes, please separate this to another PR

Reply:
Updates related to second change removed with [this commit](https://github.com/renovatebot/renovate/pull/5601/commits/032e6dbbe4580f02c6f3fad78ee148ef136d9e03). New PR opened for it [here](https://github.com/renovatebot/renovate/pull/5611).